orchagent/Makefile.am

```diff
@@ -34,7 +34,8 @@ dist_swss_DATA = \
 		 watermark_bufferpool.lua \
 		 lagids.lua \
 		 tunnel_rates.lua \
-		 trap_rates.lua
+		 trap_rates.lua \
+		 queue_ctr_rates.lua
```

Comment:
Include them in all the PRs. Also to merge, you can add the order of merge to avoid any PR merge and submodule update issue